osier/models/capacity_expansion.py

```diff
@@ -191,3 +214,12 @@ def _evaluate(self, x, out, *args, **kwargs):
 
         if self.n_constr > 0:
             out["G"] = out_constr
+
+
+        if (self.verbosity =='debug'):
```

Comment:
```suggestion
        if (self.verbosity == 'debug'):
```
Also, the `logging` module may work better here

Reply:
I'm not super familiar with the logging module, but you're probably right. You can make an issue if you'd like to see that feature.